source/index.js

```diff
@@ -1,64 +1,52 @@
-import fs from 'node:fs';
-import path from 'node:path';
 import {execa} from 'execa';
 import {deleteAsync} from 'del';
 import Listr from 'listr';
-import {merge, throwError, catchError, filter, finalize} from 'rxjs';
-import hasYarn from 'has-yarn';
+import {merge, catchError, filter, finalize, from} from 'rxjs';
 import hostedGitInfo from 'hosted-git-info';
 import onetime from 'onetime';
 import {asyncExitHook} from 'exit-hook';
 import logSymbols from 'log-symbols';
 import prerequisiteTasks from './prerequisite-tasks.js';
 import gitTasks from './git-tasks.js';
-import publish, {getPackagePublishArguments} from './npm/publish.js';
+import {getPackagePublishArguments} from './npm/publish.js';
 import enable2fa, {getEnable2faArgs} from './npm/enable-2fa.js';
 import releaseTaskHelper from './release-task-helper.js';
 import * as util from './util.js';
 import * as git from './git-util.js';
 import * as npm from './npm/util.js';
+import {findLockFile, getPackageManagerConfig, printCommand} from './package-manager/index.js';
+import handleNpmError from './npm/handle-npm-error.js';
```

Comment:
nit, but I'd like these imports to be sorted together:

```suggestion
import {getPackagePublishArguments} from './npm/publish.js';
import enable2fa, {getEnable2faArgs} from './npm/enable-2fa.js';
import handleNpmError from './npm/handle-npm-error.js';
import releaseTaskHelper from './release-task-helper.js';
import {findLockFile, getPackageManagerConfig, printCommand} from './package-manager/index.js';
import * as util from './util.js';
import * as git from './git-util.js';
import * as npm from './npm/util.js';
```

---

Related, I think this file is a little unwieldy / large, but that's mainly to do with all the tasks being defined with all the other logic. I have a local branch that separates the tasks to another file though (part of the listr2 stuff), so the mass of imports would be tuned down there